pymeasure/instruments/validators.py

```diff
@@ -140,6 +140,20 @@ def truncated_discrete_set(value, values):
     return values[-1]
 
 
+def truncated_string(value, maxlength):
```

Comment:
Is that really such a useful validator to put into core (it's only used once, and there maybe not even necessary)? I'm not sure, I'd probably make a lambda for that one occasion.
If yes, this should also get mentioned in the validator documentation.

Reply:
Certainly this can be solved with a lambda in set_process. But I think the validator provides the user with the best error message. If you don't like it in core it can certainly be moved to the dcxs.py file?

Reply:
I moved it into the dcxs code. I personally would prefer to keep it as an extra function because like this it can have a proper docstring